nemo_skills/code_execution/math_grader.py

```diff
@@ -87,6 +87,15 @@
 from math import isclose
 from typing import Union
 
+from pkg_resources import DistributionNotFound, VersionConflict, require
```

Comment:
If it's easy, can we switch to the importlib.resources api? Seems like pkg_resources is deprecated https://setuptools.pypa.io/en/latest/pkg_resources.html.

Also, let's actually fail with exception, so that it's not just warning to the user, but we stop execution if the version doesn't match. Otherwise it's too easy to miss this.

Finally, just to be safe, let's put this check inside `math_equal` function as we might accidentally import this file somewhere where we don't actually call math_equal and get the failure

Reply:
Fair enough!

Reply:
Created a function to not overcomplicate the `math_equal` function